<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Registered networks</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="NetworksList"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="GenerateFromListButton">
           <property name="text">
            <string>Create QR code for selection</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Custom</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="7" column="1">
        <widget class="QLineEdit" name="customPasswordInput">
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Wi-Fi password</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>SSID</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="customSsidInput">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>SSID</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="createQr">
           <property name="text">
            <string>Create QR code</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>NetworksList</tabstop>
  <tabstop>GenerateFromListButton</tabstop>
  <tabstop>customSsidInput</tabstop>
  <tabstop>customPasswordInput</tabstop>
  <tabstop>createQr</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>customPasswordInput</sender>
   <signal>returnPressed()</signal>
   <receiver>createQr</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>828</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customSsidInput</sender>
   <signal>returnPressed()</signal>
   <receiver>customPasswordInput</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>604</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>594</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createQr</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>create_custom_qr()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>846</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>921</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>create_custom_qr()</slot>
 </slots>
</ui>
